packages/workspace/src/core/affected-project-graph/locators/implicit-json-changes.ts

```diff
@@ -1,3 +1,5 @@
+import * as flatten from 'flat';
+import * as minimatch from 'minimatch';
```

Comment:
We should probably have this as a dependency of `@nrwl/workspace` right?

Reply:
Nice catch 👍